eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -161,25 +162,27 @@ case class ExcludeChannel(desc: ChannelDesc)
 case class LiftChannelExclusion(desc: ChannelDesc)
 // @formatter:on
 
+// @formatter:off
 case class SendChannelQuery(remoteNodeId: PublicKey, to: ActorRef, flags_opt: Option[QueryChannelRangeTlv])
-
 case object GetNetworkStats
-
 case class GetNetworkStatsResponse(stats: Option[NetworkStats])
-
 case object GetRoutingState
-
 case class RoutingState(channels: Iterable[PublicChannel], nodes: Iterable[NodeAnnouncement])
+case class GetRoutingStateStreaming(subscribeToChanges: Boolean)
+case object InitialRoutingStateEnd
+// @formatter:on
 
 // @formatter:off
 sealed trait GossipOrigin
 /** Gossip that we received from a remote peer. */
-case class RemoteGossip(peer: ActorRef) extends GossipOrigin
+case class RemoteGossip(peerConnection: ActorRef, nodeId: PublicKey) extends GossipOrigin
 /** Gossip that was generated by our node. */
 case object LocalGossip extends GossipOrigin
 
 case class Stash(updates: Map[ChannelUpdate, Set[GossipOrigin]], nodes: Map[NodeAnnouncement, Set[GossipOrigin]])
 case class Rebroadcast(channels: Map[ChannelAnnouncement, Set[GossipOrigin]], updates: Map[ChannelUpdate, Set[GossipOrigin]], nodes: Map[NodeAnnouncement, Set[GossipOrigin]])
+
+case class PeerRoutingMessage(origin: RemoteGossip, message: RoutingMessage)
```

Comment:
Not entirely sure with the moving/redefinition of `PeerRoutingMessage`, let me know what you think.